packages/now-client/src/utils/index.ts

```diff
@@ -81,17 +81,24 @@ export async function buildFileTree(
   path: string | string[],
   isDirectory: boolean,
   debug: Debug
```

Comment:
Are we going to be able to use this `debug()` command during the build?

Reply:
I was able to just pass `console.log()` to it in a local script and typescript didn't complain, so I think we can pass it basically any function that accepts a string argument.